frontend/app/package.json

```diff
@@ -89,6 +89,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-testing-library": "^5.0.1",
```

Comment:
**note:** This was already a transitive dependency at this version, this change just makes it explicit